dipy/tracking/utils.py

```diff
@@ -937,3 +939,50 @@ def reduce_rois(rois, include):
             exclude_roi |= rois[i]
 
     return include_roi, exclude_roi
+
+
+def vals_from_img(img, streamlines, affine=None):
+    """
```

Comment:
Hmm... after a bit more thinking  it seems the function needs to be renamed and moved to ``dipy.tracking.streamline``. The name of the function I propose is ``dipy.tracking.streamline.scalar_values``. This function will be useful also for @martcous who is currently preparing some PRs related to tractometry. @martcous give us a review here please and tell us if you have a different preference.